index.js

```diff
@@ -33,7 +34,9 @@ console.log('hello world')
  */
 
 // TODO: ADD CODE HERE
-
+let firstName="Nibras";
+let lastName= "Aljoubori";
+console.log("My name is " +  firstName +" "+ lastName)
```

Comment:
Looks good but when the string gets complicated you can also use  the back tick above the tab key, It would look like a regular string besides the places you insert a varible`My name is ${firstName} ${lastName}`